src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToArrayTransformer.php

```diff
@@ -72,14 +72,15 @@ public function transform($dateTime)
             ), array_flip($this->fields));
         }
 
-        if (!$dateTime instanceof \DateTime) {
-            throw new TransformationFailedException('Expected a \DateTime.');
+        // DateTimeInterface was introduced in PHP 5.5.0.
```

Comment:
it doesn't deserve a comment because we already have this logic in many other places without a comment.